### PR TITLE
add difficulty indicator to cards and update related styling

### PR DESCRIPTION
**Pull request by deadlight**
### Done
- Added difficulty to the tutorial cards. 
 - Restyled the cards to simplify them
 - updated card font colors

### QA
`polymer serve`
[Go to the homepage](http://localhost:8080)
Compare the page to [the designs](https://www.dropbox.com/s/hu4frwgqthl6pbe/Difficulty%20level.jpg?dl=0)
Check all viewports

### CODE REVIEW

File: src/elements/codelabs-card.html [hunk 1]
```diff
@@ -22,9 +23,13 @@
...
         height: 100%;
         position: relative;
+
+        --paper-card-content: {
+          padding: 0 16px 16px 16px;
+        };
       }
       paper-card a {
-        color: var(--primary-text-color);
+        color: #111;
```
Comment (WillMoggridge) [inline src/elements/codelabs-card.html]: Is this intentionally different from the primary text color, or should it be updated in `ubuntu-tutorials-app.html`?

File: src/elements/codelabs-card.html [hunk 2]
```diff
@@ -66,21 +70,13 @@
...
       }
-      @media only screen and (min-width: 768px) {
-        .card-content {
-          height: 260px;
-        }
-      }
       .meta {
-        color: var(--secondary-text-color);
-        font-size: 1em;
+        color: #666;
```
Comment (WillMoggridge) [inline src/elements/codelabs-card.html]: `--secondary-text-color` should already be set to this.

File: src/elements/difficulty-indicator.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      .difficulty-indicator__value {
+        background-image: url('https://assets.ubuntu.com/v1/69c63bcd-level+of+difficulty.png');
+        background-position-y: 5px;
+        background-repeat: no-repeat;
+        display: inline-block;
+        margin-left: 3px;
+        text-indent: -99999px;
+        width: 75px;
+      }
+
```
Comment (WillMoggridge) [inline src/elements/difficulty-indicator.html]: I will nitpick and say I think we want one empty line here and not two

File: src/elements/difficulty-indicator.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      .difficulty-indicator__value--4 {
+        background-position-x: -45px;
+      }
+
+      .difficulty-indicator__value--5 {
+        background-position-x: -60px;
+      }
+    </style>
+    <span class$="difficulty-indicator__value difficulty-indicator__value--[[difficulty]]">[[difficulty]] out of 5 will</span>
```
Comment (WillMoggridge) [inline src/elements/difficulty-indicator.html]: Didn't save this comment! Probably didn't want my name here :)

Comment (deadlight) [inline src/elements/difficulty-indicator.html]: haha!

File: src/elements/codelabs-card.html (hunk 1 above)
Comment (deadlight) [inline src/elements/codelabs-card.html]: I'll check with design

**Comment by anthonydillon** [comment]
@deadlight If we add the label design needed then we should provide QA steps for design.

**Comment by deadlight** [comment]
@anthonydillon demoing tutorials isn't possible at this stage so I need to merge the pull request so they can see on staging.

**Pull request merged**